GiniVision/Classes/CameraViewController.swift

```diff
@@ -71,11 +71,16 @@ public typealias CameraErrorBlock = (_ error: CameraError) -> ()
     fileprivate var defaultImageView: UIImageView?
     // note that in the mapping left and right are reversed. That's because landscape values have different meanings
     // in UIDeviceOrientation and AVCaptureVideoOrientation. Refer to their documentaion for more info
-    fileprivate let orientationsMapping = [UIDeviceOrientation.portrait: AVCaptureVideoOrientation.portrait,
+    fileprivate let deviceOrientationsMapping = [UIDeviceOrientation.portrait: AVCaptureVideoOrientation.portrait,
                                            UIDeviceOrientation.landscapeRight: AVCaptureVideoOrientation.landscapeLeft,
                                            UIDeviceOrientation.landscapeLeft: AVCaptureVideoOrientation.landscapeRight,
                                            UIDeviceOrientation.portraitUpsideDown: AVCaptureVideoOrientation.portraitUpsideDown]
     
+    fileprivate let interfaceOrientationsMapping = [UIInterfaceOrientation.portrait: AVCaptureVideoOrientation.portrait,
```

Comment:
Aa I told to Alpar, the deviceOrientationMapping is needed because UIInterfaceOrientation has not changed when viewWillTransition is called. And the interfaceOrientationMapping is needed because It could be the case that there is no UIDeviceOrientation (and you need it on initialization).